docs/demos/sequential_testing.ipynb

```diff
@@ -0,0 +1,3260 @@
+{
```

Comment:
<p>It's odd that so many of these show up as 0.90909. I wonder if this is a bug. It would also be nice to show at least one significantly context-dependent predictor, since all of these are p &gt; 0.05 without correcting for FDR. You might need to simulate data to show this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cnellington/Contextualized/pull/220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>